apps/firehose_to_es_processor/lib/json_object_stream.py

```diff
@@ -0,0 +1,27 @@
+import json
+
+
+class JsonObjectStream:
```

Comment:
DANGER DANGER.  Would not want to build a json parser.  Does something like https://github.com/isagalaev/ijson satisfy your requirements?

Reply:
So if you look at this closely, all it does is determine boundaries of json objects. `json.loads` is still used to actually parse the json. Does this allay your fears?

Reply:
ummm, not really?  :)

just not a fan of hand-written parsers without a buuuuunch of tests.

Reply:
I discussed this with Tony in person